Modules/posixmodule.c

```diff
@@ -8426,6 +8436,9 @@ os_kill_impl(PyObject *module, pid_t pid, Py_ssize_t signal)
     DWORD err;
     HANDLE handle;
 
+    /* Console processes which share a common console can be sent CTRL+C or
+       CTRL+BREAK events, provided they handle said events. */
+#if !defined(MS_WINDOWS_NON_DESKTOP)
```

Comment:
This should be `!defined(MS_WINDOWS_GAMES)`. 

`GenerateConsoleCtrlEvent()` is available for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM)` in Windows 10.